api/v1/client/src/commonMain/kotlin/HttpClientUtils.kt

```diff
@@ -89,8 +89,3 @@ fun createOrtHttpClient(
 
         config()
     }
```

Comment:
Doesn't this change belong in the previous commit?

Reply:
No. Below you see that this class was still being used.